src/box/lua/schema.lua

```diff
@@ -1647,9 +1647,9 @@ base_index_mt.fselect = function(index, key, opts, fselect_opts)
     -- Screen size autodetection.
     local function detect_width()
         local ffi = require('ffi')
-        ffi.cdef('void rl_get_screen_size(int *rows, int *cols);')
+    ffi.cdef('void tnt_rl_get_screen_size(int *rows, int *cols);')
```

Comment:
The indentation got broken. And below too.

Reply:
Wow, sorry, I fixed it.